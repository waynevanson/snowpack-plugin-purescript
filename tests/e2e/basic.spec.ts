import { commandSync } from "execa";
import * as path from "path";
import * as fs from "fs";

describe("basic", () => {
  it("should build", () => {
    const cwd = path.join(__dirname, "/fixtures/basic");
    commandSync("pnpm snowpack build", { cwd });
    expect(
      fs.readFileSync(path.join(cwd, "build/dist/Main.js"), {
        encoding: "utf-8",
      })
    ).toMatchSnapshot();
  });
});
