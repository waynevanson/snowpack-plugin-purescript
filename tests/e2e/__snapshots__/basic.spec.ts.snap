// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic should build 1`] = `
"// Generated by purs version 0.14.5
\\"use strict\\";
var Effect_Console = require(\\"../Effect.Console/index.js\\");
var main = Effect_Console.log(\\"\\\\ud83c\\\\udf5d\\");
module.exports = {
    main: main
};
"
`;
